### ISSUE TITLE
Cider optimization

### ISSUE THREAD

**Issue by mactavish91** [microsoft/FIBER#6]
Hello, when I run the function ```compute_caption_cider```, the rl loss has been kept at 20-80, and the cider score has not increased after training, do you know the reason?

**Comment by zdou0830** [comment]
Hi, rl loss may not be a good indicator of the model performance. if you keep the hyperparameters unchanged and train the model for 1 epoch, you should be able to get noticeable improvements in cider

**Comment by mactavish91** [comment]
> Hi, rl loss may not be a good indicator of the model performance. if you keep the hyperparameters unchanged and finetune an MLE-trained model for at least 1 epoch, you should be able to get noticeable improvements in cider

'''
rl_loss = rl_probs.view(-1) * (
            100.0 - 100.0 * torch.tensor(batch_cider_scores, device=rl_probs.device).view(-1)
        )
```
@zdou0830 Thank you very much, could you tell me why to subtract the following variable with 100.0？When I debugged, I found that the cider scores ranged from 0.6  to 1.9.

**Comment by zdou0830** [comment]
You can subtract with any reasonable baseline value and it shouldn't have much effect on the final performance.

**Comment by Markin-Wang** [comment]
Hi, thanks for your work and code. May I ask how to get or generate the 'coco-train-words.p' used in Cider optimization?

**Comment by zdou0830** [comment]
its here. I got it from the VinVL codebase https://drive.google.com/file/d/1N_O7tkBjJRCueQj7MHMqrazDTWr4OCDE/view?usp=sharing

**Comment by Markin-Wang** [comment]
> its here. I got it from the VinVL codebase https://drive.google.com/file/d/1N_O7tkBjJRCueQj7MHMqrazDTWr4OCDE/view?usp=sharing

Thanks for your quick reply, it is much helpful.
